config/am.py

```diff
@@ -0,0 +1,58 @@
+"Amharic config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-", "ʼ",
+                  "ሀ", "ሁ", "ሂ", "ሃ", "ሄ", "ህ", "ሆ",
+                  "ለ", "ሉ", "ሊ", "ላ", "ሌ", "ል", "ሎ", "ሏ",
+                  "ሐ", "ሑ", "ሒ", "ሓ", "ሔ", "ሕ", "ሖ", "ሗ",
+                  "መ", "ሙ", "ሚ", "ማ", "ሜ", "ም", "ሞ", "ሟ",
+                  "ሠ", "ሡ", "ሢ", "ሣ", "ሤ", "ሥ", "ሦ", "ሧ",
+                  "ረ", "ሩ", "ሪ", "ራ", "ሬ", "ር", "ሮ", "ሯ",
+                  "ሰ", "ሱ", "ሲ", "ሳ", "ሴ", "ስ", "ሶ", "ሷ",
+                  "ሸ", "ሹ", "ሺ", "ሻ", "ሼ", "ሽ", "ሾ", "ሿ",
+                  "ቀ", "ቁ", "ቂ", "ቃ", "ቄ", "ቅ",
+                  "ቆ", "ቈ", "ቊ", "ቋ", "ቌ", "ቍ",
+                  "በ", "ቡ", "ቢ", "ባ", "ቤ", "ብ", "ቦ", "ቧ",
+                  "ቨ", "ቩ", "ቪ", "ቫ", "ቬ", "ቭ", "ቮ", "ቯ",
+                  "ተ", "ቱ", "ቲ", "ታ", "ቴ", "ት", "ቶ", "ቷ",
+                  "ቸ", "ቹ", "ቺ", "ቻ", "ቼ", "ች", "ቾ", "ቿ",
+                  "ኀ", "ኁ", "ኂ", "ኃ", "ኄ", "ኅ",
+                  "ኆ", "ኈ", "ኊ", "ኋ", "ኌ", "ኍ",
+                  "ነ", "ኑ", "ኒ", "ና", "ኔ", "ን", "ኖ", "ኗ",
+                  "ኘ", "ኙ", "ኚ", "ኛ", "ኜ", "ኝ", "ኞ", "ኟ",
+                  "አ", "ኡ", "ኢ", "ኣ", "ኤ", "እ", "ኦ", "ኧ",
+                  "ከ", "ኩ", "ኪ", "ካ", "ኬ", "ክ",
+                  "ኮ", "ኰ", "ኲ", "ኳ", "ኴ", "ኵ",
+                  "ኸ", "ኹ", "ኺ", "ኻ", "ኼ", "ኽ",
+                  "ኾ", "ዀ", "ዂ", "ዃ", "ዄ", "ዅ",
+                  "ወ", "ዉ", "ዊ", "ዋ", "ዌ", "ው", "ዎ",
+                  "ዐ", "ዑ", "ዒ", "ዓ", "ዔ", "ዕ", "ዖ",
+                  "ዘ", "ዙ", "ዚ", "ዛ", "ዜ", "ዝ", "ዞ", "ዟ",
+                  "ዠ", "ዡ", "ዢ", "ዣ", "ዤ", "ዥ", "ዦ", "ዧ",
+                  "የ", "ዩ", "ዪ", "ያ", "ዬ", "ይ", "ዮ",
+                  "ደ", "ዱ", "ዲ", "ዳ", "ዴ", "ድ", "ዶ", "ዷ",
+                  "ጀ", "ጁ", "ጂ", "ጃ", "ጄ", "ጅ", "ጆ", "ጇ",
+                  "ገ", "ጉ", "ጊ", "ጋ", "ጌ", "ግ",
+                  "ጎ", "ጐ", "ጒ", "ጓ", "ጔ", "ጕ",
+                  "ጠ", "ጡ", "ጢ", "ጣ", "ጤ", "ጥ", "ጦ", "ጧ",
+                  "ጨ", "ጩ", "ጪ", "ጫ", "ጬ", "ጭ", "ጮ", "ጯ",
+                  "ጰ", "ጱ", "ጲ", "ጳ", "ጴ", "ጵ", "ጶ", "ጷ",
+                  "ጸ", "ጹ", "ጺ", "ጻ", "ጼ", "ጽ", "ጾ", "ጿ",
+                  "ፀ", "ፁ", "ፂ", "ፃ", "ፄ", "ፅ", "ፆ",
+                  "ፈ", "ፉ", "ፊ", "ፋ", "ፌ", "ፍ", "ፎ", "ፏ",
+                  "ፐ", "ፑ", "ፒ", "ፓ", "ፔ", "ፕ", "ፖ", "ፗ")
+
+INITIAL_PUNCTUATION = union('"', "'", "”")
+
+FINAL_PUNCTUATION = union("!", '"', ",", ".", ":", ";", "?",
+                          "፠", "፡", "።", "፣", "፤",
+                          "፥", "፦", "፧", "፨")
+
+NUMBERS = union("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
```

Comment:
Since these are always the same, do we want to define this in the normalizer as WESTERN_ARABIC_DIGITS or something?  Then we wouldn't need to include NUMBERS unless they were a special localized number system?

Reply:
to be done in a follow up, where we create a util.py for language agnostic definitions

Reply:
This `utils.py` now exists!

Reply:
Thanks!